### PR TITLE
Ignore .env file working with the Everything Bagel compose file

### PR DESCRIPTION
**Pull request by talSofer**

### CODE REVIEW

File: deployments/compose/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.env
```
Comment (arielshaqed) [inline deployments/compose/.gitignore:1]: Maybe put this in a git ignore for the entire repo?

**Review by arielshaqed — APPROVED** [review]
Cool, very useful!

File: deployments/compose/.gitignore (hunk 1 above)
Comment (talSofer) [reply deployments/compose/.gitignore:1]: I've considered that but some parts of our repo rely on .env files. for example https://github.com/treeverse/lakeFS/blob/master/test/spark/.env

WDYT?

Comment (arielshaqed) [reply deployments/compose/.gitignore:1]: Hmmm... so maybe _not_ rely on the default file, but pass that explicitly with a CLI flag?

Comment (nopcoder) [reply deployments/compose/.gitignore:1]: I also think we need to switch to use .env in the repo level and fix the spark test not to use it.

**Review by nopcoder — APPROVED** [review]
Approved because it is serve what we wanted

File: deployments/compose/.gitignore (hunk 1 above)
Comment (talSofer) [reply deployments/compose/.gitignore:1]: There are many other places that use .env files. let me look at it

Comment (arielshaqed) [reply deployments/compose/.gitignore:1]: > There are many other places that use .env files. let me look at it

I'm saying essentially that I would like us **not** to use this default.  It is suitable for interactive use, less so for committed code.  If a docker-compose uses an env file and is stored in the VCS, then that file is liable to be checked in.  At the very least it should not be a hidden file.

So I _do_ consider ignoring all `.env` files (the docker-compose default name) part of this task.  And I further believe that we should rename all other `.env` -> `env`, and use the [`--env-file` flag](https://docs.docker.com/compose/env-file/) to refer to that file _explicitly_.

Comment (talSofer) [reply deployments/compose/.gitignore]: Done

**Review by nopcoder — APPROVED** [review]
LGTM

**Pull request merged**